src/transformState.js

```diff
@@ -5,7 +5,36 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const type = action.type; // string with type property
+    const addValues = action.extraData; // object with extra data
+    const removeValues = action.keysToRemove; // array with remove keys
```

Comment:
use object destructuring
```suggestion
    const { type, extraData, keysToRemove} = action;
```